### DISCUSSION TITLE
Please add a `tail-comments:{number}` filter

### DISCUSSION THREAD

**Discussion by gz-jgale** [community/community#52545] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

More often than not, I need to search for keywords and mentions in the most recent comment on an Issue in a Project view, my user issues list, and in my notifications. 

It would be super helpful to have a new filter named something like `tail-comments:{number}` that would accept an integer value to filter search results to only the last `{number}` comments from each issue. This could behave similar to how the [unix tail command](https://www.unix.com/man-page/Linux/1/tail/) works on lines of text from a file or stream.

Using the [`last-updated:{NUMBER}days`](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-for-when-an-item-was-last-updated), and [`iteration modifiers`](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-number-date-and-iteration-fields) do not always help because I would like to see all comments that contain a string or username regardless of when the last comment happened.

Thank you for considering my feature request!

**Comment by opegamanalitarok** [comment]
This is an interesting feature request! Unfortunately, at this time, there isn't a built-in filter in GitHub that allows you to search only the last {number} comments on an issue. However, there are a few workarounds you could try:

1. Use the in:comments qualifier: You can use the "in:comments" qualifier to search only the comments of issues, pull requests, or commits. For example, to search for the keyword "example" in the last 10 comments of issues, you can use the following search query: "example in:comments updated:>2022-01-01"

2. Use the API: You can use the GitHub API to search for issues and their comments. You could use the "List issue comments for a repository" API endpoint and filter the comments based on the issue ID and the date they were created or updated. Here's the link to the API documentation: https://docs.github.com/en/rest/reference/issues#list-issue-comments-for-a-repository

I hope this helps!🙌🙌
